services/ui-src/src/utils/reports/entities.ts

```diff
@@ -61,6 +116,17 @@ export const getFormattedEntityData = (
       break;
 
     case ModalDrawerEntityTypes.QUALITY_MEASURES:
+      entityData = {
+        domain: getRadioValue(entity, "qualityMeasure_domain"),
+        name: entity?.qualityMeasure_name,
+        nqfNumber: entity?.qualityMeasure_nqfNumber,
+        reportingRateType: getReportingRateType(entity),
+        set: getRadioValue(entity, "qualityMeasure_set"),
+        reportingPeriod: getReportingPeriod(entity),
+        description: entity?.qualityMeasure_description,
+        numberOfPlans: reportFieldData?.plans.length,
+        plans: entity ? getPlans(entity, reportFieldData) : undefined,
```

Comment:
Note: With the aforementioned changes we'll probably simplify this to just be something like (pseudocode warning):

```js
Object.entries(entity) // to array
.filter(**down to just the ones that start with whatever_something_measure_**) // filter time
.fromEntries() || undefined // back to obj or something idk i'm tired
```